static/css/src/_base.scss

```diff
@@ -121,3 +122,73 @@ a {
         color: var(--link-txt--visited);
     }
 }
+
+/*
+Blockquote
+
+Markup:
+<blockquote>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In bibendum ante purus, nec gravida elit iaculis id.
+Morbi rhoncus lacus quam, ut consectetur lorem dapibus et. Phasellus vel cursus tellus.</blockquote>
+
+Styleguide Base.Blockquote
+*/
+
+blockquote {
+    position: relative;
+    margin: 0;
+    padding-inline: 16px;
+    color: #97AAB4;
+    font-style: italic;
+
+    &::before {
+        content: "";
+
+        display: block;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 4px;
+        height: 100%;
+        background-color: #A7B7C0;
+    }
+}
+
+/*
+Ordered list
+
+Markup:
+<ol type="1">
+    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+    <li>Aliquam facilisis lacus ex, ac egestas velit maximus et. </li>
+    <li>Vivamus suscipit erat nec nibh pharetra efficitur. </li>
+    <li>Integer et fringilla mi. </li>
+    <li>Donec dolor sem, vehicula et massa ac, vestibulum vulputate odio. Donec sed leo vitae libero commodo facilisis.</li>
+</ol>
+
+Styleguide Base.OrderedList
+*/
+
+ol {
+    margin: 0;
+    padding: 0;
+}
```

Comment:
Missing the spacing between `<li>` elements.

> List spacing: 6px

![image](https://github.com/user-attachments/assets/6455c37a-a96e-4440-b423-fabc9899959f)